<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>Загрузка графика</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parametresButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_docx">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>docx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Экспорт графика&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_pdf">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>pdf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_png">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>png</string>
    </property>
   </widget>
   <widget class="QPushButton" name="notepad">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>Блокнот</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>701</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="technical_indikactor">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <item>
     <property name="text">
      <string>Скользящая средняя(SMA)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Относительный индекс силы(RSI)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Скользящая средняя затрат(EMA)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Балансовый объем(OBV)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="choiceButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>Установить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>260</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Цвет</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="font_size">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>260</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="brushButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>130</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>Режим рисования</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cloze">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>Цена закрытия</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="min">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>Цена минимума</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="max">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>Цена максимума</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="open">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>Цена открытия</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>170</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(250, 165, 37)</string>
       </property>
       <property name="text">
        <string>Кисть</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">choiceGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(250, 165, 37)</string>
       </property>
       <property name="text">
        <string>Линейка</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">choiceGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>701</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="value_tech">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 165, 37)</string>
    </property>
    <property name="text">
     <string>Сбросить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="choiceGroup"/>
 </buttongroups>
</ui>
